### PR TITLE
From master to main transition

### PR DESCRIPTION
**Pull request by cosama**
This pull request will fix the internals so that we can rename master to main and remove develop.

Changes on github required to go along with this pull request:

- [x] Create a new branch `main` from head of `master` and push
- [x] Change default branch to `main`
- [x] Change this pull requests origin to `main`
- [x] Change all other pull request to `main`
- [ ] Merge pull request to `main`
- [ ] Remove `develop` and `master`

Let me know if I forgot something.

### CODE REVIEW

**Comment by jccurtis** [comment]
Status of those TODO's?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -62,4 +62,4 @@ Use [google standards](https://google.github.io/styleguide/pyguide.html)
     - `pylint becquerel tests`
   - [ ] Spellcheck your code and docstrings
   - [ ] Check style is [consistent with Google Python style guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md)
-  - [ ] Push branch to GitHub and create a pull request
```
Comment (markbandstra) [inline CONTRIBUTING.md:65]: `main`

File: RELEASING.md [hunk 2]
```diff
@@ -2,20 +2,17 @@
 
 We follow the `git flow` [release process](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow).
 
-- [ ] Pull the most recent versions of `develop` and `master`
```
Comment (markbandstra) [inline RELEASING.md:5]: "most recent version"

**Comment by cosama** [comment]
The TODO's is what needs to be done simultaneously with the pull request. Once I start messing around with those the repo will probably be non functional until it is done.

File: RELEASING.md [hunk 3]
```diff
@@ -2,20 +2,17 @@
...
+- [ ] Pull the most recent versions of `main`
+- [ ] Branch off of `main` and name the branch `release-X.X.X` or `hotfix-X`
 - [ ] Update version number within the repository
   - in `setup.py`
   - in Copyright Notice in `README`
   - in the `LICENSE`
 - [ ] Update classifiers in `setup.py`
 - [ ] Verify that all tests pass (`python setup.py test`)
-- [ ] Commit the changes, push to GitHub, and start a pull request into `master`
-- [ ] Approve PR and merge it into master, but do not delete it yet
```
Comment (markbandstra) [inline RELEASING.md:17]: `main`

Comment (markbandstra) [reply RELEASING.md:17]: Also below

Comment (cosama) [reply RELEASING.md:17]: Those is the old version, it has been all changed to main?

File: CONTRIBUTING.md (hunk 1 above)
Comment (cosama) [reply CONTRIBUTING.md:65]: Yeah added 'to main' in the new version.

File: RELEASING.md (hunk 3 above)
Comment (jccurtis) [reply RELEASING.md:17]: Yeah this is on the neg diff. The pos diff below is right

Comment (markbandstra) [reply RELEASING.md:17]: Weird, my comments went on the old code, supposed to be on the new code. Just noticed a few places where main was in ticks and a few places it wasn't, just pushing for them all to be consistently `main`.

Comment (markbandstra) [reply RELEASING.md:17]: My bad, I clicked on the wrong side of the line

File: RELEASING.md [hunk 4]
```diff
@@ -2,20 +2,17 @@
...
-- [ ] Create tagged version (`X.X.X`) on GitHub
+- [ ] Commit the changes, push to GitHub, and start a pull request into `main`
+- [ ] Approve PR, merge it into main, and delete release or hotfix branch
+- [ ] Create tagged version (`X.X.X`) on GitHub pointing to the merge commit to main
```
Comment (jccurtis) [inline RELEASING.md:15]:
```suggestion
- [ ] Create tagged version (`X.X.X`) on [GitHub](https://github.com/lbl-anp/becquerel/releases/new) pointing to the merge commit to main
```

File: RELEASING.md (hunk 2 above)
Comment (markbandstra) [reply RELEASING.md:5]: This comment also meant for the new code, not the old code.

File: .travis.yml [hunk 5]
```diff
@@ -13,8 +13,7 @@ install:
 script: "python setup.py test"
 branches:
     only:
-        - master
-        - develop
+        - main
 notifications:
   slack:
     secure: fbsAu4oLvRWKGpPLzJcjobftOK1bOjmvjkrJbQ2+QcrpWqhsU0oSrBr4812zOQBbtU3LkyAgOYOMeKtIqkxaUaZed+73Lv69ZpdBpzx0tY9Oj5Qm/sUYd8g2NjYnK3GpbSp3rK0CVMIhVAopM14GYqWTAoCLCeyDw5jNnx9gLW0AQN35dozKH0kEdp/q2kMt3vsd8/hauJjtiYsBzlIh4s+92qZiYgKkHzqvX27KUfwWNXve18QxYKlzldApVtmstm0J2o+tV8Wzx3x9FBSWJK40pXjVT/KEdWEdCLnh85ZtzEdRc7sRThDBKxQBuGkGm78a4pD5fj7imM7eVR9mQK9+nl1JJ/BOVwIz1ij7kF9Kl8WnGZkfU0W5eRwaJMXHv9ohP3yXXccloAjCHPS+Dr9nGeFNQKuGSTj2Pd9NDPhKh350hB1qA4cIu9uerrcHZEaSab0Ethc5xHPELZQXPmnthuPPlRSoCTaO9G2RKp07KdiTaC6UGUw9rkzaJEmTQU9Xzcpog3AdO/cBVQ7EMsWyPSQqn8vk9QO8Sj47h009IE4UzBbkfnDW2QYzv0LOgC8IvAdfuL6yo2ljATqFz9ppbPGnNEQA+FpBiuJKO/yI8k2NPLtKKQ5G9xPVriXwD0OMeMGLQk+yRrXmF3bOZ40b8eNpH/8AkwV1QgYsKBU=
```
Comment (jccurtis) [inline .travis.yml:19]: Side note, this should be moved to a travis variable ...

File: RELEASING.md (hunk 3 above)
Comment (cosama) [reply RELEASING.md:17]: oh the ticks :+1:

File: .travis.yml (hunk 5 above)
Comment (jccurtis) [reply .travis.yml:19]: Nevermind I'm thinking about gitlab ... 🤦

Comment (markbandstra) [reply .travis.yml:19]: How do we do that?

**Comment by cosama** [comment]
I think I addressed all the comments. Please approve if you think it is all good and you agree with the process outlined in the TODO's.

File: .travis.yml (hunk 5 above)
Comment (jccurtis) [reply .travis.yml:19]: Not needed

**Review by jccurtis — APPROVED** [review]
Thanks @cosama , please proceed when ready.

**Comment by cosama** [comment]
The transition is now completed. `main` just successfully build on travis. I think all is working.

**Pull request merged**